### PR TITLE
feat: Create footer

### PR DESCRIPTION
**Pull request by Manavpatel2005**

### CODE REVIEW

File: src/Pages/Festival/constant.ts [hunk 1]
```diff
@@ -1,9 +1,9 @@
-import Janmashtami from "../../assets/image/janmashtami.jpg";
-import RathYatra from "../../assets/image/rathYatra.jpg";
-import Holi from "../../assets/image/holi.jpg";
-import Radhashtami from "../../assets/image/radhashtami.jpg";
-import GovardhanPuja from "../../assets/image/govardhanPuja.jpg";
-import KartikDipawali from "../../assets/image/kartikDipawali.jpg";
+import Janmashtami from "../../assets/images/janmashtami.jpg";
```
Comment (vaidehi27) [inline src/Pages/Festival/constant.ts]:
```suggestion
import janmashtamiImage from "../../assets/images/janmashtami.jpg";
```

File: src/Pages/Festival/constant.ts [hunk 2]
```diff
@@ -1,9 +1,9 @@
...
+import RathYatra from "../../assets/images/rathYatra.jpg";
```
Comment (vaidehi27) [inline src/Pages/Festival/constant.ts]:
```suggestion
import rathYatraImage from "../../assets/images/rath-yatra.jpg";
```

File: src/Components/Footer/index.tsx [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+  MDBCol,
+  MDBIcon,
+} from "mdb-react-ui-kit";
+import footerLogo from "../../assets/images/footer-logo.jpeg";
+import { FooterLogoStyle, ContactStyle, BgColor, TimeStyle } from "./style";
+import { contactInfos, schedules } from "./constant";
+
+export const Footer = () => {
+  return (
+    <MDBFooter className="text-center text-lg-start text-muted  ">
```
Comment (vaidehi27) [inline src/Components/Footer/index.tsx]:
```suggestion
    <MDBFooter className="text-center text-lg-start text-muted">
```

File: src/Pages/FooterTerms/CancellationPolicy/index.tsx [hunk 4]
```diff
@@ -0,0 +1,8 @@
+import { FooterTerms } from "../PageLayout/index";
```
Comment (vaidehi27) [inline src/Pages/FooterTerms/CancellationPolicy/index.tsx]: `PageLayout` is a component so it should be within `../components/PageLayout`

**Review by vaidehi27 — APPROVED** [review]

File: src/Pages/FooterTerms/CancellationPolicy/index.tsx (hunk 4 above)
Comment (Priyansh1825) [reply src/Pages/FooterTerms/CancellationPolicy/index.tsx]: done

**Pull request merged**